### ISSUE TITLE
New Marlin fw incompatible?

### ISSUE THREAD

**Issue by frollard** [daid/LegacyCura#63]
Hi again;

I picked up Cura windows RC2 and giddily installed last night; noting one major change;

First run wizard: I upgraded the firmware in case there was any major changes.  The old firmware shipped with RC1 was around 42kb, the new one is 80kb.  With RC2 update, the first run wizard during sanity checks get to 'Disabling Step motors' then echoes, "the unit did not reply properly to the m084, check temperature wires" (or something along those lines).  It will not continue through the heater/xyz checks.  I downgraded to the firmware that came with RC1 and it calibrates fine in RC2 after that - so RC2 works fine, just the firmware packaged with it has something funky. 

Win7 64
Cura RC2 (and 1)
UM with 2560 board

First run>
Update firmware (successful)
Run Sanity Checks (fail at disabling motors)
<skip>
Extruder calibrate (fails) - known issue

Without calibration, I didn't attempt to print anything on the RC2 packaged firmware.

**Comment by daid** [comment]
Dammit. That's what I get for not properly testing all features with the new firmware. Explains some of the troubles we noticed at the UltiEvening.

The RC2 firmware is larger because it contains the UltiPanel+SD features. Maybe that's what's causing the problem. I included those features because I was pretty sure Ultimaker was going to release the UltiPanel the same day: https://shop.ultimaker.com/en/parts-1/ulticontroller-ultipanel.html

It might be timing out because it's waiting for the SD card that never responds...
